lib/response.js

```diff
@@ -788,14 +789,7 @@ res.location = function location(url) {
     loc = String(url);
   }
 
-  var m = schemaAndHostRegExp.exec(loc);
-  var pos = m ? m[0].length + 1 : 0;
-
-  // Only encode after host to avoid invalid encoding which can introduce
-  // vulnerabilities (e.g. `\\` to `%5C`).
-  loc = loc.slice(0, pos) + encodeUrl(loc.slice(pos));
-
-  return this.set('Location', loc);
+  return this.set('Location', encodeUrl(loc));
```

Comment:
can't recall now if we want this or not

Reply:
I would need to look through the change log in more detail to tell for sure, but I think not based on this one: https://github.com/expressjs/express/commit/da4d763ff6ba9df6dbd8f1f0b1d05412dda934d5#diff-c03f99ad1a5cf68a9b32468371973a5898c4c9c9d5f097e124f1d2b7b27db11dR924